webrtc.html

```diff
@@ -5829,10 +5829,17 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
       <div>
         <pre class="idl">dictionary RTCRtpCodecCapability {
              DOMString mimeType;
+             unsigned long  clockRate;
+             unsigned short channels = 1;
+             DOMString      sdpFmtpLine;
 };</pre>
         <section>
           <h2>Dictionary <dfn>RTCRtpCodecCapability</dfn>
           Members</h2>
+          <p>The <code>RTCRtpCodecCapability</code> dictionary provides
+          information about codec capabilities. Only capability combinations
+          utilizing a distinct payload type in the corresponding SDP
+          are provided.</p>
```

Comment:
I think this wording could be improved, since if `getCapabilities` is called before any offer/answer is created, there isn't any corresponding SDP yet.

Maybe something like:

> Only capability combinations that would utilize distinct payload types in a generated SDP offer are provided.

Or, to use less SDP-specific terminology:

> Exactly one `RTCRtpCodecCapability` will be present for each supported combination of parameters that must be identical between the offer and answer.

Some examples may also be helpful. For example:

* Exactly one Opus codec, with "stereo=1", indicating that the browser supports either stereo or mono.
* Two H.264 codecs, one for each packetization mode.
* Two CN codecs with different clock rates